OpenEdXMobile/res/values/dimens.xml

```diff
@@ -65,11 +65,16 @@
     <dimen name="login_logo_margin_bottom">15dp</dimen>
     <dimen name="login_social_divider_margin_top">20dp</dimen>
     <dimen name="login_social_divider_margin_bottom">15dp</dimen>
+    <dimen name="divider_with_text_margin">12dp</dimen>
```

Comment:
@BenjiLee I told you yesterday that I didn't think it belonged with the other divider dimens. But I fixed it to appease you. Now I put it iust below the only 2 instances that say "dimen" and you don't like it. Where would you like me to put it?